extensions/extension_mgr.go

```diff
@@ -251,6 +256,9 @@ func (e *set) ToProto() ([]*extensions.SimpleExtensionURI, []*extensions.SimpleE
 		})
 	}
 
+	// Sort extensions by the anchor for consistent output
+	sort.Slice(uris, func(i, j int) bool { return uris[i].ExtensionUriAnchor < uris[j].ExtensionUriAnchor })
```

Comment:
This is not strictly necessary, but seemed nicer: it's consistent, which is nice; it's in order of anchors, which also seems nice; and it makes testing easier, because its deterministic